app/code/Magento/Sales/Api/InvoiceCancelInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Sales\Api;
+
+/**
+ * Interface InvoiceCancelInterface
+ *
+ * @package Magento\Sales\Api
+ */
+interface InvoiceCancelInterface
+{
+
+    /**
+     * Cancel invoice
+     *
+     * @param int $invoiceId
+     * @return bool
+     */
+    public function cancel($invoiceId);
```

Comment:
Since this PR is targeting `2.2-develop` it is possible to declare scalar type hint and return type for the method.